src/parsing/tokens.rs

```diff
@@ -150,7 +150,7 @@ impl<'a> TokenFinder<'a> {
 
     #[inline]
     fn get_first_token_before(&self, node: &dyn Spanned, is_match: impl Fn(&TokenAndSpan) -> bool) -> Option<&'a TokenAndSpan> {
-        node.previous_tokens_fast(self.module).find(|token_and_span| is_match(token_and_span))
+        node.previous_tokens_fast(self.module).iter().find(|token_and_span| is_match(token_and_span))
```

Comment:
Hmmm... I should add a test that catches this. Right now should be `.iter().rev()` as I changed it to keep everything it returns in file order (ex. `get_previous_siblings()` returns in file order, then people can reverse it via `.iter().rev()` if they need to). Will open an issue and I fixed this in master already.